### PR TITLE
feat(#770): code() in OpcodeName.java for compatibility with opeo

### LINKED ISSUE #770: `OpcodeName#code` for compatibility with opeo
Currently, `org.eolang.jeo.representation.directives.OpcodeName` does not support `code()` method, which is used by 
[opeo](https://github.com/objectionary/opeo-maven-plugin) `org.eolang.opeo.ast.OpcodeName` inside `it/AgentsIT.java`. Let's add this method to `org.eolang.jeo.representation.directives.OpcodeName`in order to resolve https://github.com/objectionary/opeo-maven-plugin/issues/75

### PR DESCRIPTION
**Pull request by h1alexbel**
In this pull I've added new method to `OpcodeName.java`: `code()`. This method is required to be used by [OPEO](https://github.com/objectionary/opeo-maven-plugin) in order to resolve https://github.com/objectionary/opeo-maven-plugin/issues/75.

closes #770
History:
- **feat(#770): opcode**
- **feat(#770): no empty**
- **feat(#770): puzzle**
- **feat(#770): typo**
- **feat(#770): should**

---

## PR-Codex overview
This PR introduces a new method `code()` in the `OpcodeName` class and adds a corresponding test in `OpcodeNameTest`. The method retrieves the opcode value, and the test verifies its functionality.

### Detailed summary
- Added a Javadoc comment for the `code()` method in `OpcodeName`.
- Implemented the `code()` method to return the opcode value.
- Created a new test method `returnsOpcode()` in `OpcodeNameTest` to validate the `code()` method's output.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by h1alexbel** [comment]
@volodya-lombrozo take a look, please

File: src/test/java/org/eolang/jeo/representation/directives/OpcodeNameTest.java [hunk 1]
```diff
@@ -53,6 +54,22 @@ void checksOpcodeNames(final int actual, final String expected) {
...
     }
 
+    @Test
+    void returnsOpcode() {
+        final int expected = 0;
+        final int code = new OpcodeName(expected, OpcodeNameTest.COUNTER)
+            .code();
+        MatcherAssert.assertThat(
+            String.format(
+                "Opcode code: %s does not matches with expected: %s",
```
Comment (volodya-lombrozo) [inline src/test/java/org/eolang/jeo/representation/directives/OpcodeNameTest.java:64]: @h1alexbel NIT: it's better to use `%d` for numbers.  [Docs.](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Formatter.html)

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]
@h1alexbel Thank you for the contribution! Only one small correction is needed.

**Comment by h1alexbel** [comment]
@volodya-lombrozo updated pull request. Take a look again, please

**Comment by volodya-lombrozo** [comment]
@h1alexbel Looks good t me, thank you.

**Review by volodya-lombrozo — APPROVED** [review]

**Comment by volodya-lombrozo** [comment]
@rultor merge

**Comment by h1alexbel** [comment]
@volodya-lombrozo could you release a new version of JEO?

**Pull request merged**